src/test/java/at/ac/tuwien/kr/alpha/test/util/TestUtils.java

```diff
@@ -95,4 +117,78 @@ public static Atom basicAtomWithSymbolicTerms(String predicate, String... consta
 		return new BasicAtom(pred, trms);
 	}
 
+	private static SystemConfig regressionTestConfigToSystemConfig(RegressionTestConfig cfg) {
```

Comment:
If I remember correctly, then we came to the conclusion to move this method to `RegressionTestConfig`. Was there a reason not to move it?